docs/en-US/component/select-v2.md

```diff
@@ -183,6 +199,15 @@ select-v2/use-valueKey
 | loading-text | 远程加载时显示的文字 | string | — | 加载中 | -->
 </span>
 
+## props
```

Comment:
```suggestion
## props ^(2.4.1)
```

Reply:
Do you think this version number is necessary? @tolking

Reply:
I think adding a version tag here is not conducive to expanding attributes in the future.

Reply:
Makes sense.